content/workshops/django-end-to-end-testing.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Effective End-to-End Testing for Django
+sub: A Workshop
```

Comment:
Is this missing the rest of the line? 
![Screenshot 2023-10-23 at 3 27 54 PM](https://github.com/oddbird/oddleventy/assets/1581694/5cb352f1-9749-4481-9572-d2ef2f59d429)